satnogsclient/web/static/css/app.css

```diff
@@ -0,0 +1,6 @@
+body {
+  padding-top: 50px;
+}
+.settings {
+  padding: 40px 15px;
+}
```

Comment:
If we want to have unified code style we could go with 4 spaces in css. See [Network's editorconfig](https://github.com/satnogs/satnogs-network/blob/dev/.editorconfig).